app/src/main/java/com/github/se/studybuddies/navigation/Destinations.kt

```diff
@@ -18,4 +18,4 @@ val BOTTOM_NAVIGATION_DESTINATIONS =
         Destination(route = Route.SOLOSTUDYHOME, icon = R.drawable.user_v2, textId = "Solo study"),
         Destination(route = Route.GROUPSHOME, icon = R.drawable.groups, textId = "Groups"),
         Destination(route = Route.CHAT, icon = R.drawable.messages, textId = "Messages"),
-        Destination(route = Route.SOLOSTUDYHOME, icon = R.drawable.map, textId = "Map"))
+        Destination(route = Route.TODOLIST, icon = R.drawable.map, textId = "Map"))
```

Comment:
These destinations are used for the navigation bar ^^' You shouldn't modify them. The ToDoList Destination should be add in the Route.kt file and then linked to the corresponding Button Icon in the SoloStudy mode. 
Your modification could be used for testing but not for the final version (and even the later version as the map will be implemented)